google/cloud/bigtable/admin/integration_tests/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(bigtable_admin_integration_tests
 
 include(CreateBazelConfig)
 export_list_to_bazel("bigtable_admin_integration_tests.bzl"
-                     "bigtable_admin_integration_tests")
+                     "bigtable_admin_integration_tests" YEAR 2021)
```

Comment:
Nit: My version of this also tweaked other instances, where about 75% of the time we double-quote the year.

Reply:
Good point, I think this is now fixed across all files. PTAL